Bottom: ead4a19316ea90f45b643ab625170c80fb7951ac
Top:    df587a9462d674d0c6657853a41fef4caeb729e4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-02-19 09:10:55 +0000

Refresh of validate-and-swap-from-to

---

diff --git a/src/action.c b/src/action.c
index e3b0192..ab4a822 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1302,8 +1302,8 @@ NotifyMode (void)
 		  layer_from = GetLayerNumber (PCB->Data, lastLayer);
 		  layer_to = GetLayerNumber (PCB->Data, CURRENT);
 
-                  layer_from = MIN (last_layer_num, current_layer_num);
-                  layer_to =   MAX (last_layer_num, current_layer_num);
+		  layer_from = MIN (last_layer_num, current_layer_num);
+		  layer_to =   MAX (last_layer_num, current_layer_num);
 		}
 	      else
 		{
